contract/src/orca.contract.js

```diff
@@ -62,125 +96,36 @@ export const meta = {
 harden(meta);
 
 /**
- * @param {ZCF} zcf
+ * @typedef {{
+ *   localchain: Remote<LocalChain>;
+ *   orchestrationService: Remote<CosmosInterchainService>;
+ *   storageNode: Remote<StorageNode>;
+ *   timerService: Remote<TimerService>;
+ *   agoricNames: Remote<NameHub>;
+ * }} OrchestrationPowers
+ */
+
+/**
+ * @param {ZCF<Record<string, unknown>> | undefined} zcf
```

Comment:
This seems equivalent to the default...

```suggestion
 * @param {ZCF} zcf
```